distributed/client.py

```diff
@@ -1568,30 +1568,8 @@ def _graph_to_futures(self, dsk, keys, restrictions=None,
 
         return futures
 
-    @gen.coroutine
-    def _get(self, dsk, keys, restrictions=None, loose_restrictions=None,
-             resources=None, raise_on_error=True):
-        futures = self._graph_to_futures(dsk, set(flatten([keys])),
-                restrictions, loose_restrictions, resources=resources)
-
-        packed = pack_data(keys, futures)
-        try:
-            result = yield self._gather(packed)
-        except Exception as e:
-            if raise_on_error:
-                raise
-            else:
-                result = 'error', e
-                raise gen.Return(result)
-        finally:
-            for f in futures.values():
-                f.release()
-        if not raise_on_error:
-            result = 'OK', result
-        raise gen.Return(result)
```

Comment:
I don't know of anyone who uses this except for internal code.  We might want to include an informative error?  No strong thoughts really.